scripts/deploy-on-kind.sh

```diff
@@ -47,9 +47,42 @@ EOF
   kind export kubeconfig --name "$cluster" --kubeconfig "$HOME/.kube/$cluster.yml"
 }
 
+ensure_local_registry() {
+  helm repo add twuni https://helm.twun.io
+  helm upgrade --install localregistry twuni/docker-registry --set service.type=NodePort,service.nodePort=30050,service.port=30050
+
+  # TODO-maybe we don't need to add the /etc/hosts hack if we configure the mirror below to redirect to 127.0.0.1?
```

Comment:
Instead of adding TODOs in the code I would suggest having a chore to look into configuring the mirror